HHS/Head-Start-TTADP
# [TTAHUB-1765] Complete event page for the training report form

## Description of change
Add complete event page to the training report form. Add submission handler and multi-level validations.

## How to test
Add some sessions in the database. Make some complete and some "In progress" (the only valid statuses for sessions).
Note that they appear on the complete event page and that having "in progress" sessions does not allow you to "complete" a training report. 

Change all sessions to complete in the database and confirm that you can't submit a report unless you have all fields complete

Confirm that you can submit a form once all sessions are complete and all fields are filled out. 

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1765


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
